doc/api/deprecations.md

```diff
@@ -40,7 +40,7 @@ However, the deprecation identifier will not be modified.
 
 ## List of deprecated APIs
 
-<a id="DEP0001"></a>
+<i id="DEP0001"></i>
 ### DEP0001: `http.OutgoingMessage.prototype.flush`
```

Comment:
```suggestion
### <span id="DEP0001">DEP0001: <code>http.OutgoingMessage.prototype.flush</code></span>
```
&nbsp;
@aduh95, what do you think about this?

Reply:
That could work, maybe we could wrap only the DEP code, to be able to keep the back ticks instead of `<code>` tags:

```suggestion
### <span id="DEP0001">DEP0001</span>: `http.OutgoingMessage.prototype.flush`
```

I don't know if we should care about that, but it does add about 1KiB of data to the markdown file…

Reply:
Even better! If the Markdown is rendering this properly then that's all we need to get this taken care of IMO.

Reply:
OK so I just tried that, unfortunately this approach doesn't work: The `<span>` gets copied into the TOC, and that's not right as now there are two elements with the same ID… The anchor cannot be part of the heading.

Reply:
We had this same problem w/ `<code>` tags appearing in each document title.

I don't recall how that got fixed, but hopefully, we can use the same remedy for this problem as well.

/cc @Trott who probably knows the answer :)

Reply:
It's very likely that the way the TOC is being generated needs to be re-evaluated.

I think this is the PR I was thinking of, but perhaps not. I think there was another one that sanitized the HTML tags.

https://github.com/nodejs/node/pull/31086

Reply:
OK interesting 🤔 but maybe that could be done in a following PR once the TOC generation script has been amended?

FWIW, FontAwesome is also using empty `<i>` tags and here is their rationale ([source](https://fontawesome.com/how-to-use/on-the-web/referencing-icons/basic-use)):

> We like the `<i>` tag for brevity and because most folks use `<em></em>` for emphasized/italicized semantic text these days. If that’s not your cup of tea, using a `<span>` is more semantically correct.